src/Models/OnlineMailable.php

```diff
@@ -2,11 +2,20 @@
 
 namespace Sammyjo20\Wagonwheel\Models;
 
+use Database\Factories\OnlineMailableFactory;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
 use Illuminate\Database\Eloquent\Model;
 
 class OnlineMailable extends Model
 {
+    use HasFactory;
+
     protected $casts = [
         'expires_at' => 'timestamp',
     ];
+
+    protected static function newFactory()
+    {
+        return new OnlineMailableFactory;
```

Comment:
I think we should be using the static `new()` method instead of instantiating ourselves, safer for future versions of Laravel that might add some functionality here.

Reply:
I was having trouble with the namespacing of the Model factory, that's why I made the call to `newFactory()`. 

Without this method, Laravel tries to load the factory from a non existent namespace. 
How would you address this issue? 

```
Error: Class 'Database\Factories\Sammyjo20\Wagonwheel\Models\OnlineMailableFactory' not found
```

Reply:
Yeah, using this method is fine. I was suggesting `OnlineMailableFactory::new()` instead of `new OnlineMailableFactory`, this is the same thing that Laravel does under the hood.

Reply:
Ah, check 👍

Reply:
@ryangjchandler updated in commit 6dcefd4